core/base64.js

```diff
@@ -56,9 +56,7 @@ export default {
     ],
     /* eslint-enable comma-spacing */
 
-    decode(data, offset) {
-        offset = typeof(offset) !== 'undefined' ? offset : 0;
-
+    decode(data, offset = 0) {
```

Comment:
Default arguments are not supported in internet explorer. I expect that this gives an error in IE

Reply:
Perhaps babel handles this?

Reply:
Yeah. Babel handles this.
There was a similar discussion in some other PR and there are already some default arguments in the codebase.